04/index.html

```diff
@@ -1,30 +1,60 @@
 <!DOCTYPE html>
 <html lang="pl">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./styles/global.css" />
+    <link
+      rel="stylesheet"
+      href="./styles/mobile.css"
+      media="(max-width: 700px)"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/desktop.css"
+      media="(min-width: 701px)"
+    />
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
-</head>
-<body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
+  </head>
+  <body>
+    <nav class="navbar">
+      <div class="navbar__logo">
+        <img
+          class="navbar__logo-img"
+          src="./assets/logo.png"
```

Comment:
Uwaga na wielkość liter. Plik graficzny ma zapisane rozszerzone WIELKIMI literami. Na systema UNIX (linux, macos) ma to znaczenie.